### PR TITLE
Add SECURITY.md

### LINKED ISSUE #14: Add a default SECURITY.md
With https://github.com/jenkinsci/.github/blob/master/SECURITY.md as content/example

### PR DESCRIPTION
**Pull request by NotMyFault**
When I was creating the hacktoberfest stats repository, I noticed, that GH now proactively nags you about adding those files.

For now, I copied what the jenkinsci org uses: https://github.com/jenkinsci/.github/blob/master/SECURITY.md
I'd assume, the information, how to use the private issue tracker could be used here too.

I'd like to request @jenkins-infra/security for a review.

Fixes #14

### CODE REVIEW

**Comment by dduportal** [comment]
We might want to update it a bit more: @wadeck (current security officer) and I (current Infra officer) discussed that the infrastructure security would be managed by the infra officer as first responder, who can escalate to security team if required.

With this split of concerns, that need we might want to update this content a bit.


Anyway, thanks for raising this!

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
+To that end, we work with Jenkins core and plugin developers, as well as security researchers, to fix security vulnerabilities in Jenkins in a timely manner, and to improve the security of Jenkins in general.
```
Comment (Wadeck) [inline SECURITY.md:5]:
```suggestion
We make every possible effort to ensure users can adequately secure their automation infrastructure.
To that end, we want to guarantee that the infrastructure part of the project is as secure as possible, to keep the supply chain safe to build and deliver Jenkins components.
```

**Review by Wadeck — APPROVED** [review]
Good idea Alex 👍 

I suggested a small change to be accurate and also created a PR on the reporting page to ensure we accept vulnerability reports on infra components (https://github.com/jenkins-infra/jenkins.io/pull/5602)

**Review by dduportal — APPROVED** [review]
Thanks folks! LGTM

**Pull request merged**